code/game/machinery/doors/brigdoors.dm

```diff
@@ -63,6 +63,24 @@
 			stat |= BROKEN
 		update_icon()
 
+/obj/machinery/door_timer/proc/print_report()
+	var/logname = input(usr, "Name of the guilty?","[id] log name")
+	var/logcharges = input(usr, "What are they being charged with?","[id] log charges")
+
+	for(var/obj/machinery/computer/prisoner/C in prisoncomputer_list)

```

Comment:
Did you add all files to this commit / PR?

prisoncomputer_list isn't defined anywhere, and doesn't populate anywhere.